lib/MetaCPAN/Web/Controller/Sysadmin/Datacenters.pm

```diff
@@ -0,0 +1,29 @@
+package MetaCPAN::Web::Controller::Sysadmin::Datacenters;
+
+use Moose;
```

Comment:
It's actually our critic config that sucked.  I've now made it suck less via ae6573d6973